Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -64,6 +67,15 @@ ChromeDriver CreateDriverInner(ChromeDriverService service)
 
             options.SetCommonChromiumOptions(configuration);
 
+            // While the Edge driver easily locates Edge on Windows, it struggles on Linux, where the different release
+            // channels have different executable names. This setting looks up the "microsoft-edge-stable" command and
+            // sets the full path as the browser's binary location.
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
+                CliWrapHelper.WhichAsync("microsoft-edge-stable").GetAwaiter().GetResult().FirstOrDefault() is { } binaryLocation)
```

Comment:
Shouldn't it rather throw if `binaryLocation` is `null`, i.e. `microsoft-edge-stable` failed?

Reply:
I don't think so, because the driver may still resolve it correctly.

Reply:
OK then.